PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -620,7 +620,16 @@ def miniAOD_customizeData(process):
     from PhysicsTools.PatAlgos.tools.coreTools import runOnData
     runOnData( process, outputModules = [] )
     process.load("RecoCTPPS.Configuration.recoCTPPS_cff")
+    process.load('L1Trigger.L1TGlobal.simGtExtFakeProd_cfi')
+    process.simGtExtFakeProd.bxFirst = cms.int32(-2)
+    process.simGtExtFakeProd.bxLast = cms.int32(2)
+    process.simGtExtFakeProd.setBptxAND = cms.bool(True)
+    process.simGtExtFakeProd.setBptxPlus = cms.bool(True)
+    process.simGtExtFakeProd.setBptxMinus = cms.bool(True)
+    process.simGtExtFakeProd.setBptxOR = cms.bool(True)
+    process.simGtExtFakeProd.tcdsRecordLabel = cms.InputTag("tcdsDigis","tcdsRecord")
     task = getPatAlgosToolsTask(process)
+    task.add(process.simGtExtFakeProd)
```

Comment:
Not sure I understand correctly, or I missed the discussion with @slava77 @silviodonato @qliphy 

Should this line be under
(run2_miniAOD_UL).toModify(....)
instead. So that it will apply only when run2_miniAOD_ul is activated.

Reply:
indeed, this is a backport and here we need to use the UL modifier

Reply:
Thanks @srimanob.   You are probably right.

Reply:
adjusting now, and testing wfs 136.7611 and 4.53